addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureGitScanRule.java

```diff
@@ -68,6 +69,10 @@
             Pattern.compile(
                     "<html"); // helps eliminate some common false positives in the case of 403s,
     // 302s, etc
+    private static final Map<String, String> ALERT_TAGS =
+            CommonAlertTag.toMap(
+                    CommonAlertTag.OWASP_2021_A05_SEC_MISCONFIG,
+                    CommonAlertTag.OWASP_2017_A03_DATA_EXPOSED);
```

Comment:
The source code disclosure rules should probably have the same tags, whichever ones we agree on...

Reply:
The ones that have specific CVEs I'm going to leave a vuln component ok?